articles/azure-monitor/toc.yml

```diff
@@ -47,6 +47,8 @@ items:
       href: app/tutorial-app-dashboards.md
     - name: Monitor mobile or UWP application
       href: app/mobile-center-quickstart.md
+    - name: Application Insights for ASP.NET Core applications
```

Comment:
Lets make it the 1st one under Application Insights (given other tutorials are likely out-of-date)

Reply:
Fixed